_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: default
+title: Mutate event
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Mutate event processors
+
+You can use mutate event processors to add entries to an event, delete entries from an event, rename keys in an event, copy values within an event, and convert value types in an event. <!---Need a better overall description to explain what they do in general.--->
```

Comment:
@oeyh What do mutate event processors allow you to do as a whole? Modify events? Why is this useful to them?

Reply:
Mutate event processors allow you to modify events in Data Prepper. Events are representation of data that flows through Data Prepper, so this processor provide users a general method to edit the data representation according to their use cases.